omod/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ProviderController2_0Test.java

```diff
@@ -141,17 +141,16 @@ public void doSearch_shouldProcessIncludeAll() throws Exception {
 
 			// Replay
 			List<?> allProviders = deserialize(handle(request)).get("results");
-			List<Object> allUuids = ((List<Map>) allProviders).stream().map(
-					p -> p.get("uuid")
-			).collect(Collectors.toList());
 
-			// Verify: 'includeAll=true' same as Java API
+            // Verify: 'includeAll=true' same as Java API
 			Assert.assertArrayEquals(
 					Context.getProviderService().getProviders(providerName, null, null, null, true)
 							.stream().map(
 							BaseOpenmrsObject::getUuid
-					).toArray(),
-					allUuids.toArray()
+					).sorted().toArray(),
```

Comment:
Why are we now sorting?

Reply:
@dkayiwa If we don't sort the elements, the order in the two arrays will differ, which causes the test to fail.

Reply:
How was it passing before without your sorting changes?

Reply:
Can we make the test do the comparison in a way that does not require sorting?

Reply:
I updated this test